src/doc.ts

```diff
@@ -0,0 +1,9 @@
+export type Doc = {
+  '@id': string
+};
```

Comment:
As memux doesn't require things to be jsonld docs, we maybe should put this type else (context?). Could potentially be done later.